php/src/Communicator.cpp

```diff
@@ -57,22 +55,20 @@ class ActiveCommunicator : public IceUtil::Shared
     int expires;
     std::chrono::steady_clock::time_point lastAccess;
 };
-typedef IceUtil::Handle<ActiveCommunicator> ActiveCommunicatorPtr;
+using ActiveCommunicatorPtr = shared_ptr<ActiveCommunicator>;
```

Comment:
I'd be inclined to remove these aliases. Though maybe in a followup PR?

Reply:
I am more inclined to embrace these Ptr aliases.

Reply:
let's do in a separate PR